setup.py

```diff
@@ -8,15 +8,22 @@
     long_description = fp.read()
 
 setup(
-    name='Gibberish',
+    name='gibberish',
```

Comment:
I was gonna suggest again `gibberish-yplan` but now I've seen that it's super old code I'm not worried about name squatting